setup.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+
+import os
+import sys
+from setuptools import setup
+
+
```

Comment:
This is pretty bare-bones, but it works fine!

I might well replace it with a more complex one I have from another project after this review is done.  Or, not.  :-)